### ISSUE TITLE
AttributeError: module 'scvelo.tl' has no attribute 'differential_kinetic_test'

### ISSUE THREAD

**Issue by amunzur** [theislab/scvelo#189]
Hi Volker, 

I am not able to use the following scVelo functions: 

```
scv.pl.proportions()
scv.tl.differential_kinetic_test()
scv.tl.score_genes_cell_cycle()
scv.utils.get_cell_transitions()
scv.get_df()
```

For each of them, I am getting the same errors: 

```
AttributeError: module 'scvelo.tl' has no attribute 'differential_kinetic_test'
AttributeError: module 'scvelo.tl' has no attribute 'score_genes_cell_cycle'
AttributeError: module 'scvelo.utils' has no attribute 'get_cell_transitions'
AttributeError: module 'scvelo.pl' has no attribute 'proportions'
AttributeError: module 'scvelo' has no attribute 'get_df'

```
I can use all the other functions just fine, just the 5 above is problematic. 
These are versions I have: 
```
scvelo==0.1.25  scanpy==1.4.6  anndata==0.7.1  loompy==3.0.6  numpy==1.18.1  scipy==1.4.1  matplotlib==3.1.3  sklearn==0.23.0  pandas==1.0.3 
```
Can you please help? Maybe I am missing some packages?

**Comment by VolkerBergen** [comment]
Those modules are only available since the latest release v0.2, so you'd need to `pip install -U scvelo`.

**Comment by VolkerBergen** [comment]
`scv.logging.print_version()` should also give you a warning, that there's a new version available.

**Comment by chen-peng-1874** [comment]
> Those modules are only available since the latest release v0.2, so you'd need to `pip install -U scvelo`.

I already updated scvelo version, but it did not work.
Is there any other reasons? Could it be the envs directories' problem?

**Comment by WeilerP** [comment]
@chen-peng-1874, I just checked and everything works for me. So the issue must be with your installation/environment setup.
